static-config-parts/static-webpack-config.js

```diff
@@ -1,105 +0,0 @@
-// SURGEON GENERAL'S WARNING: THIS IS NOT A WEBPACK CONFIG, THIS IS A FUNCTION
-// THAT ENHANCES THE BASE REACT-STATIC WEBPACK CONFIG SO WE CAN USE POSTCSS
-
-const ExtractTextPlugin = require("extract-text-webpack-plugin");
-const autoprefixer = require("autoprefixer");
-
-const staticWebpackConfig = (config, { defaultLoaders, stage }) => {
-  const preciousDarlingPostCSSPlugins = [
-    {
-      test: /\.css$/,
-      use: [
-        {
-          loader: "css-loader",
-          options: {
-            importLoaders: 1,
-            minimize: stage !== "dev",
-            sourceMap: stage === "dev"
-          }
-        },
-        {
-          loader: "postcss-loader",
-          options: {
-            sourceMap: false,
-            ident: "postcss",
-            plugins: [
-              require("postcss-flexbugs-fixes"),
-              require("postcss-import"),
-              require("postcss-browser-reporter"),
-              require("postcss-reporter"),
-              require("postcss-preset-env"),
-              require("postcss-focus"),
-              require("postcss-custom-media")({ stage: 0 }),
-              require("postcss-url")({ url: "inline", encodeType: "base64" }),
-              autoprefixer({
-                browsers: [
-                  ">1%",
-                  "last 4 versions",
-                  "Firefox ESR",
-                  "not ie < 9" // React doesn't support IE8 anyway
-                ],
-                flexbox: "no-2009"
-              })
-            ]
-          }
-        }
-      ]
-    },
-    {
-      test: /\.(svg)(\?.*)?$/,
```

Comment:
I think the svg loader may still be required. When running `yarn build` and `yarn serve` on this branch, svgs are not rendering:
<img width="1424" alt="Screen Shot 2020-01-05 at 3 36 49 PM" src="https://user-images.githubusercontent.com/3719995/71787836-3873b980-2fd1-11ea-8bf5-505a2e164f7f.png">